lib/timeline/TimeStep.js

```diff
@@ -578,6 +578,7 @@ TimeStep.prototype.getLabelMinor = function(date) {
       if(this.isMajor() && date.weekday() !== 0){
           return "";
       }
+      break;
```

Comment:
You're really sure this is not an intentional fall-through? (timeline agnostic reviewing here)